src/contributing.md

```diff
@@ -189,7 +189,10 @@ changes. Every local commit that touched the subtree causes a commit on the remo
 modified to move the files from the specified directory to the tool repo root.
 
 Make sure to not pick the `master` branch on the tool repo, so you can open a normal PR to the tool
-to merge that subrepo push.
+to merge that subrepo push. It is also recommended that the branch 
+you use as the push target should either be a brand new branch, 
+or first delete the branch if targeting an existing branch, 
+because changes cannot be fast forwarded.
```

Comment:
The "either... or..." in the second sentence is little confusing. Maybe something like this would work?

```suggestion
to merge that subrepo push. It is also recommended that the branch 
you use as the push target should either be a brand new branch, 
or a branch that exists on the remote.
If the latter, you will need to delete the branch locally, because changes cannot be fast forwarded.
```

Reply:
It's not just the local branch, remote needs to be deleted as well